src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -23,12 +24,14 @@
 use Psr\Container\ContainerInterface as PsrContainerInterface;
 use Psr\Http\Client\ClientInterface;
 use Psr\Log\LoggerAwareInterface;
+use Rize\UriTemplate;
```

Comment:
```suggestion
use Rize\UriTemplate as RizeUriTemplate;
```

Reply:
I would consider this to be a bug in the fixer: The fixer should detect that a class with the same name has already been imported from a different namespace and either do nothing or generate an alias for the import. Currently, the fixer would generate obviously broken code.

Reply:
@derrabus yes, I agree, obviously it's a bug in the importing logic. I overlooked it in my PR 🙁.

Reply:
Actually there was a test case for names collision, but it involved existing import + new symbol, while the problem here was related to names collision between newly found symbols, that I did not take into consideration. I've prepared fix for that in PHP-CS-Fixer/PHP-CS-Fixer#7635 🙂.